curriculum/challenges/english/01-responsive-web-design/accessibility-quiz/part-023.md

```diff
@@ -0,0 +1,139 @@
+---
+id: 6143931a113bb80c45546287
+title: Part 23
+challengeType: 0
+dashedName: part-23
+---
+
+# --description--
+
+Arguably, `D.O.B.` is not descriptive enough. This is especially true for visually impaired users. One way to get around such an issue, without having to add visible text to the label, is to add text only a screen reader can read.
+
+Append a `span` element with a class of `sr-only` to the current text content of the third `label` element.
+
+# --hints--
+
+You should add a `span` element within the third `label` element.
+
+```js
+assert.exists(document.querySelector('label:nth-of-type(3) > span'));
```

Comment:
This syntax doesn't seem to work, presumably because the `label` elements aren't siblings.

Reply:
This was silly of me. Thank you, for catching it.